docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -0,0 +1,83 @@
+Based on historical data, the {classification} analysis predicts whether it is 
+safe or risky to lend money to a given loan applicant. In the second case, the 
+data we have represents songs and the analysis – based on the features of the 
+data points – classifies the songs as hip-hop, country, classical, or any 
+other genres available in the set of categories we have. Therefore, 
+{classification} is for predicting discrete, categorical values, unlike 
+{reganalysis} which predicts continuous, numerical values.
+
+From the perspective of the possible output, there are two types of 
+{classification}: binary and multi-class {classification}. In binary 
+{classification} the variable you want to predict has only two potential values. 
+The loan example above is a binary {classification} problem where the two 
+potential outputs are `safe` or `risky`. The music classification problem is an 
+example of multi-class {classification} where there are many different potential 
+outputs; one for every possible music genre. In the {version} version of the 
+{stack}, you can perform binary {classanalysis}.
+
+
+[discrete]
+[[dfa-classification-features]]
+==== {feature-vars-cap}
+
+When you perform {classification}, you must identify a subset of fields that you 
+want to use to create a model for predicting another field value. We refer to 
+these fields as _{feature-vars}_ and _{depvar}_, respectively. 
+{feature-vars-cap} are the values that the {depvar} value depends on. There are 
+three different types of {feature-vars} that you can use with our 
+{classification} algorithm: numerical, categorical, and boolean. Arrays are not 
+supported in the {feature-var} fields.
+
+
+[discrete]
+[[dfa-classification-supervised]]
+==== Training the {classification} algorithm
+
+{classification-cap} – just like {regression} – is a supervised {ml} process. It 
+means that you need to supply a labeled training dataset that has some 
+{feature-vars} and a {depvar}. The {classification} algorithm learns the 
+relationships between the features and the {depvar}. Once you’ve trained the 
+model on your training dataset, you can reuse the knowledge that the model has 
+learned about the relationships between the data points to classify new data.
+
+
+[discrete]
+[[dfa-classification-model]]
+===== {classification-cap} models
+
+The ensemble algorithm that we use in the {stack} is a type of boosting called 
+boosted tree regression model which combines multiple weak models into a 
+composite one. We use decision trees to learn to predict the probability that 
+a feature belongs to a certain class.
+
+
+[discrete]
+[[dfa-classification-evaluation]]
+==== Measuring model performance
+
+You can measure how well the model has performed on your dataset by using the 
+`classification` evaluation type of the 
+{ref}/evaluate-dfanalytics.html[evaluate {dfanalytics} API]. The metric that the 
+evaluation provides you is the multi-class confusion matrix which tells you how 
```

Comment:
Not sure if "multi-class" should be mentioned since we don't support it in 7.5.

Reply:
Good point. I talked to Sophie about it and we agreed that it is okay to mention until we make it clear that right now the only available classification is binary classification (as we did it in line 26-27).